heartbeat/podman-etcd

```diff
@@ -414,21 +434,21 @@ prepare_env() {
 	NODEIP="$(attribute_node_ip get)"
```

Comment:
Just add `IPURL=$(ip_url $NODEIP)` here, and use $IPURL instead of calling `ip_url()` multiple times.

Reply:
Great suggestion, I went ahead and updated the code, please give it another look over, thanks!